Collection/src/Database.java

```diff
@@ -0,0 +1,81 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+
+class Database <T extends Print> implements Print {
+    private static int b = 10;
```

Comment:
1) it is hard to understand what 'b' means, you should provide more meaningful names to quickly understand your own code in the future
2) it is better to make it constant if you are not going to change the value - private static final int
3) constant names should be uppercase by convention https://study.com/academy/lesson/java-naming-conventions-variables-constants.html#:~:text=Convention%3A%20Use%20Uppercase%20with%20Underscore,underscore%20character%20separating%20the%20words.